alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/accessor/ProviderTaskPropertiesAccessor.java

```diff
@@ -0,0 +1,12 @@
+package com.synopsys.integration.alert.common.persistence.accessor;
+
+import java.util.Optional;
+
+import com.synopsys.integration.alert.common.exception.AlertDatabaseConstraintException;
+
+public interface ProviderTaskPropertiesAccessor {
+    Optional<String> getTaskProperty(String taskName, String propertyKey);
```

Comment:
Should the getTaskProperty also take the configId?  Even though the task name is unique because it currently has the config id in the task name.